vividus-plugin-parquet/src/main/resources/properties/defaults/run.properties

```diff
@@ -0,0 +1 @@
+expression.csv-to-parquet.csv-format=T(org.apache.commons.csv.CSVFormat).DEFAULT
```

Comment:
it doesn't look really user-friendly, any ideas how to simplify `T(org.apache.commons.csv.CSVFormat).DEFAULT` -> `DEFAULT`